### PR TITLE
Fix links on Languages & Frameworks

### PR DESCRIPTION
**Pull request by fknipp**
This will also add Deno.

### CODE REVIEW

File: src/docs/languages-and-frameworks.md [hunk 1]
```diff
@@ -19,5 +19,6 @@ Below is a list of language and framework specific tips & tricks.
 - [Julia](/docs/languages/julia/)
 - [LaTeX](/docs/languages/latex/)
 - [R](/docs/languages/r/)
-- [Kotlin](/docs/kotlin-in-gitpod/)
-- [Pandas](/docs/frameworks/pandas/)
+- [Kotlin](/docs/languages/kotlin)
```
Comment (gtsiolis) [inline src/docs/languages-and-frameworks.md:22]: **praise**: Nice catch on both entries @fknipp!

File: src/docs/menu.ts [hunk 2]
```diff
@@ -237,7 +237,15 @@ export const MENU: MenuEntry[] = [
...
                 "Kotlin",
                 "languages/kotlin"
-            )
+            ),
+            M(
+                "Pandas",
+                "languages/python/#pandas"
+            ),
+            M(
+                "Deno",
```
Comment (gtsiolis) [inline src/docs/menu.ts:246]: **nitpick**: Also added Deno on the navigation on the left sidebar.

File: src/docs/menu.ts [hunk 3]
```diff
@@ -237,7 +237,15 @@ export const MENU: MenuEntry[] = [
             M(
                 "Kotlin",
                 "languages/kotlin"
-            )
+            ),
+            M(
+                "Pandas",
```
Comment (gtsiolis) [inline src/docs/menu.ts:242]: **note**: Added pandas here to match the existing links in the parent _Languages & Frameworks_ page.

**Review by gtsiolis — APPROVED** [review]
Thanks for contributing @fknipp! 🎉 

Let's merge this!

File: src/docs/languages-and-frameworks.md [hunk 4]
```diff
@@ -19,5 +19,6 @@ Below is a list of language and framework specific tips & tricks.
...
+- [Pandas](/docs/languages/python/#pandas)
+- [Deno](/docs/languages/deno)
```
Comment (gtsiolis) [inline src/docs/languages-and-frameworks.md:24]: **note**: Moved Deno below to keep a track of what's been added until we redesign this page to be more useful.

**Comment by fknipp** [comment]
Thanks for fixing the other files as well, I've missed them.

**Pull request merged**